src/Apcu/bootstrap.php

```diff
@@ -13,18 +13,61 @@
     return;
 }
 
+if (extension_loaded('Zend Data Cache')) {
+    // Zend Data Cache refuses to load in the CLI, regardless of the configuration
+    if ('phpdbg' !== PHP_SAPI && 'cli' !== PHP_SAPI && false !== ini_get('zend_datacache.apc_compatibility')) {
+        require_once __DIR__.DIRECTORY_SEPARATOR.'bootstrap_zend.php';
```

Comment:
instead of a separate bootstrap file, we should use the same strategy as other polyfills (see e.g. https://github.com/symfony/polyfill/blob/master/src/Iconv/bootstrap.php): a single bootstrap file that delegates "complex" implementations to static methods.